lib/chutney/TorNet.py

```diff
@@ -763,7 +765,32 @@ def _genRouterKey(self):
                   .format(" ".join(cmdline), stdouterr))
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
-
+        
+    def _setEd25519Id(self):
+        """Read the ed25519 identity key for this router, and set up the 'ed25519-id' entry in the Environ"""
+        datadir = self._env['dir']
+        key_file = os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
+        EXPECTED_ED25519_FILE_SIZE = 64
+        CURRENT_FILE_SIZE = os.stat(key_file).st_size
+        if not os.path.exists(key_file):
+            print("File {} does not exist. Are you running a very old tor version?".format(key_file))
+            return
```

Comment:
You could either set this to `return 0` or raise an Error here to make it clear what is going on, in my opinion.

Reply:
Thanks, but we don't want to raise an error here. It is ok to run chutney with an old version of Tor.

And don't want to return 0, because the rest of the function doesn't return anything. It just sets a value.

But you're right, we need some explanation here:
```suggestion
            # If there is no file, we do not set ed25519-id
            return
```

Reply:
There is a bit more explanation on the old pull request:
 https://github.com/torproject/chutney/pull/61#discussion_r395404692